user_docs/en/changes.md

```diff
@@ -2,48 +2,90 @@
 
 ## 2025.1
 
+This release introduces NVDA Remote, allowing you to control a remote computer running NVDA from another device running NVDA.
+
+There are many improvements to speech particularly responsiveness with SAPI 4, SAPI 5 and OneCore voices.
+Rate boost and automatic language switching is now supported in SAPI 5.
+SAPI 4 voices now support audio ducking, leading silence trimming, and keeping the audio device awake.
+
+The Add-on Store automatic update system has been improved, allowing you to select channels for automatic updates, and run automatic updates in the background.
+
+It's now easier to manually refresh OCR and toggle automatic refresh, with new commands.
+
+Native selection is now available in Chrome and Edge.
+
+Microsoft Office and LibreOffice support has improved, notably with more shortcuts being announced.
+
+Default input and output braille tables can now be determined based on the NVDA language.
+Native support has been added for the Dot Pad tactile graphics device from Dot Inc as a multiline braille display.
+There have been other minor fixes for braille displays particularly around connectivity.
+
+LibLouis braille translator, eSpeak-NG and Unicode CLDR have been updated.
+New Coptic, compact Cuneiform, and Portuguese Computer Braille tables are available.
```

Comment:
```suggestion
New Coptic, compact Cuneiform, and 6 dots Portuguese Computer Braille tables are available.
```

Reply:
We use "6 dot" not dots more consistently

```suggestion
New Coptic, compact Cuneiform, and Portuguese 6 dot Computer Braille tables are available.
```